src/app/components/ATIAnalytics/params/genericPage/buildParams.ts

```diff
@@ -11,27 +11,35 @@ export const buildPageATIParams = ({
   const { atiAnalyticsAppName, atiAnalyticsProducerId, service, lang } =
     serviceContext;
   const {
-    analytics: {
-      contentId,
-      contentType,
-      pageIdentifier,
-      timePublished,
-      timeUpdated,
-    } = {},
-    title: pageTitle,
+    contentId,
+    contentType,
+    pageIdentifier,
+    timePublished,
+    timeUpdated,
+    pageTitle,
   } = atiData;
+  // const {
+  //   atiAnalytics: {
+  //     contentId,
+  //     contentType,
+  //     pageIdentifier,
```

Comment:
Needed?

Reply:
it's not. Thanks for catching this.
This had been commented out as part of running tests following a rewrite
to reference `atiAnalytics` properties.